script/utils/start-backing-services

```diff
@@ -1,40 +1,30 @@
 #!/bin/sh
 
-echo "==> Starting the backing services in Docker..."
-
 for arg in "$@"
 do
   if [[ $arg == "--mock-api" ]]; then
     MOCK_API=true
+    DOCKER_PARAM="--scale=hmpps-accredited-programmes-api=0 --scale=hmpps-accredited-programmes-db=0"
   fi
 done
 
-if [ "$MOCK_API" = true ]; then
-  docker compose up --scale=hmpps-accredited-programmes-api=0 --scale=postgresql=0 -d
+echo "==> Starting the backing services in Docker..."
+docker compose up $DOCKER_PARAM -d
 
-  echo "==> Stubbing API endpoints..."
+npx ts-node --transpile-only ./wiremock/scripts/resetStubs.ts > /dev/null
 
-  npx ts-node --transpile-only ./wiremock/scripts/resetStubs.ts > /dev/null
+if [ "$MOCK_API" = true ]; then
+  echo "==> Stubbing API endpoints..."
```

Comment:
Nice one. We can probably update this `echo` to `Stubbing Accredited Programme API endpoints...` for clarity

Reply:
Is this all it's stubbing, as the file is called stubApis.ts?

Reply:
Yep - it's probably also worth either (1) breaking down the `stubApis` script by entity, like the other scripts, or (at least) (2) renaming `stubApis` to `stubApi` or the mouthful `stubCoursesOfferingsParticipationsReferrals` (and updating the `console.log` near the end). Happy for that to be a separate piece of work if it's unclear!

Reply:
Not touching it as part of this, it's down the rabbit hole.